Chill_Yield.Rmd

```diff
@@ -57,13 +57,13 @@ knitr::write_bib(c(.packages(),
 Often data is limited 
 
 1. Lack of data
-1. How can we go about making assessments when we hve limited data?
-1. when attempting to assess the relationships between temperature and yield
-1. perhaps this data limitation does not need to hinder our assessment
-1. Maybe a lot of data is not necessary
-1. embrace uncertainty
-1. preciseness is pointless
-1. far into the future we have no data. We necesarily have uncertainty in our model forecasts
+2. How can we go about making assessments when we have limited data?
+3. when attempting to assess the relationships between temperature and yield
```

Comment:
Numbers are generated automatically in markdown. Just listing '1.' 1.' 1.' results in in '1.' '2.' '3.'